src/elli_test.erl

```diff
@@ -21,7 +21,7 @@
 call(Method, Path, Headers, Body, Opts) ->
     Callback     = proplists:get_value(callback, Opts),
     CallbackArgs = proplists:get_value(callback_args, Opts),
-    Req = elli_http:mk_req(Method, {abs_path, Path}, Headers,
+    Req = elli_http:mk_req(Method, {abs_path, Path}, Headers, Headers,
```

Comment:
would it make sense to add `mk_req/10` for the case when `Headers == ParsedHeaders`?

Reply:
or maybe `{Headers, ParsedHeaders}` instead?

Reply:
Eh, they never will, they only do here because it is a test.